packages/storybook/stories/va-accordion-uswds.stories.tsx

```diff
@@ -1,428 +1,263 @@
 import { useState, useEffect } from 'react';
 import {
   getWebComponentDocs,
-  componentStructure,
   propStructure,
   StoryDocs,
 } from './wc-helpers';
 
 const accordionDocs = getWebComponentDocs('va-accordion');
-const accordionItem = getWebComponentDocs('va-accordion-item');
+const accordionItemDocs = getWebComponentDocs('va-accordion-item');
+const accordionItemProps = propStructure(accordionItemDocs);
+
+const defaultArgs = {
+  itemsBordered: JSON.parse(accordionItemProps.bordered.defaultValue.value),
+  itemsLevel: JSON.parse(accordionItemProps.level.defaultValue.value),
+  itemsHeadlineSlot: false,
+  itemsIcon: false,
+};
+
+const defaultValues ={
+  open: JSON.parse(accordionItemProps.open.defaultValue.value),
+  subheader: JSON.parse(accordionItemProps.subheader.defaultValue.value),
+}
+
+const items = [
+  {
+    ...defaultValues,
+    id: 'first',
+    header: 'First Amendment',
+    body: 'Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances.',
+  },
+  {
+    ...defaultValues,
+    id: 'second',
+    header: 'Second Amendment',
+    body: 'A well regulated Militia, being necessary to the security of a free State, the right of the people to keep and bear Arms, shall not be infringed.',
+  },
+  {
+    ...defaultValues,
+    id: 'third',
+    header: 'Third Amendment',
+    body: 'No Soldier shall, in time of peace be quartered in any house, without the consent of the Owner, nor in time of war, but in a manner to be prescribed by law.',
+  },
+]
 
 export default {
   title: 'Components/Accordion USWDS',
   id: 'uswds/va-accordion',
-  subcomponents: componentStructure(accordionItem),
   parameters: {
     componentSubtitle: 'va-accordion web component',
     docs: {
       page: () => <StoryDocs storyDefault={Default} data={accordionDocs} />,
     },
   },
+  argTypes: {
+    ...propStructure(accordionDocs),
+    i18n: {
+      name: 'Add internationalization buttons',
+      description: 'Adds buttons to change the language of the component',
+      control: { type: 'boolean' },
+      table: { category: 'Additional variants' },
+    },
+    accordionItems: {
+      name: 'va-accordion-item: items',
+      description: 'Content and props for the accordion items',
```

Comment:
This is really just a nitpick but I'm wondering if it's possible to keep the parent "Properties" section at the top of the properties table?

![Screenshot 2025-05-28 at 10 11 12 AM](https://github.com/user-attachments/assets/da5e8e28-310a-43fd-9cab-d58a43c5cddd)